packages/template-retail-react-app/package.json

```
@@ -20,6 +20,7 @@
     "lint:fix": "npm run lint -- --fix",
     "postinstall": "npm run compile-translations && npm run compile-translations:pseudo",
     "push": "npm run build && pwa-kit-dev push",
+    "push:ssr-source-map": "cross-env PWA_KIT_SSR_SOURCE_MAP=true npm run build && pwa-kit-dev push",
```

```suggestion
    "push:ssr-source-map": "cross-env PWA_KIT_SSR_SOURCE_MAP=true npm run push",
```
This actually makes me wonder, though, what is the point of having a specific npm script for this? Why not just document the feature and let people set the environment variable when they want to use it?

Another idea: We could add a flag to `pwa-kit-dev build`, copying the pattern for the [buildDirectory flag](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/b8f77c86ddd6b396325e3f3dc2a7d081846759a8/packages/pwa-kit-dev/bin/pwa-kit-dev.js#L261-L268).

---

agreed with Will on point of having this script.

---

@wjhsf @alexvuong Will get back to this once I have approval on a new tech proposal from the platform MRT team.